conf/000-default.conf

```diff
@@ -0,0 +1,42 @@
+# $Id$
```

Comment:
This conf file is for the Apache file and it also override the `000-default.conf`.
It defines the some hosts are as follows:

- wildcard listens `443` port and the original host listens `443` port via HTTPS. They're same as listening `80` port via HTTP.
- The `ServerName` assumes that it's `ldphp.tk`.